moveit_ros/perception/lazy_free_space_updater/include/moveit/lazy_free_space_updater/lazy_free_space_updater.h

```diff
@@ -55,6 +56,8 @@ class LazyFreeSpaceUpdater
 private:
 #ifdef __APPLE__
   typedef std::unordered_map<octomap::OcTreeKey, unsigned int, octomap::OcTreeKey::KeyHash> OcTreeKeyCountMap;
+#elif __cplusplus >= 201103L
```

Comment:
I don't have anything against this, but what's the reason you decided to make this change in this PR? Does `tr1::unordered_map` not iterate correctly?

Reply:
I came across `tr1::unordered_map` when I was looking for the declaration of the map. `tr1::unordered_map` iterates well. I just try to modernize it. But I'm not quite sure about the difference between `std::unordered_map` and `std::tr1::unordered_map`, so I would like to keep the old one.